test/worlds/joint_controller.sdf

```diff
@@ -313,5 +313,84 @@
         <i_gain>0.01</i_gain>
       </plugin>
     </model>
+
+    <model name="joint_controller_test3">
+      <pose>0 0 0.005 0 0 0</pose>
```

Comment:
I'd suggest changing the pose so that this model doesn't start in collision with `joint_controller_test`.